### ISSUE TITLE
Bug: "Combine into volume" loses reading position

### ISSUE THREAD

**Issue by glottisfaun0000** [TortugaPower/BookPlayer#1131]
## Steps to reproduce the behavior
1. Begin listening to a folder of files
2. Files are marked as listened after completion
3. Combine the folder into a volume from home screen
4. All tracks in the new volume are marked as unlistened

If doing this can't transfer the listened value to the tracks, it should at least warn you. Can't be the only one who lost a position this way.

**Comment by GianniCarlo** [comment]
@glottisfaun0000 good point about the warning. Just FYI that we keep track of time different for a volume, meaning that if you switch back, you can inspect the individual items and see where you were and then switch back and skip ahead. 

More specifically, we associate the current time progress to the folder item for volumes (like if that was the book), and for regular books it's tied to the individual item
